### DISCUSSION TITLE
Displaying photos

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#7590] [Help] [answered]
Does soemone have any JS cool stuff for displaying photos ?
For example : 
_ zoom in/out capability
_ magnifying glasses
...

CC @mielune

**Comment by melohagan** [comment, marked as answer]
Custom components out soon! 😄 

You'll be able to use that to import existing libraries, such as this one I found: https://www.npmjs.com/package/svelte-magnifier/v/0.0.1

**Reply by NicolasFabre81** [reply]
Great news !
